components/overflow-group/overflow-group.js

```diff
@@ -97,6 +97,9 @@ class OverflowGroup extends OverflowGroupMixin(LitElement) {
 			::slotted(d2l-selection-action-dropdown) {
 				margin-inline-end: 0.2rem;
 			}
+			::slotted(*:last-child) {
```

Comment:
I guess the `*` is needed to increase the specificity so this overrides the selectors above?

Reply:
Like I could use `::slotted(:last-child)` if you'd prefer? Having the `*` there just makes it make sense in my head. I can't do like `::slotted(d2l-button:last-child), ...` with a list of them all since that would target the _last_ of each of them, not the last of all of them.

Reply:
Does `::slotted(> :last-child)` actually make more sense here?

Reply:
Never mind, it's not even a thing!

Reply:
> Like I could use `::slotted(:last-child)` if you'd prefer?

In terms of the checks the browser does, I suspect they are the same. I was more just wondering if you found it necessary for selectivity.

> Does ::slotted(> :last-child) actually make more sense here?

Worth noting that `::slotted` does not support CSS combinators (`>`, `~`, `+`, etc).

Reply:
Both seem to work.

Reply:
Looked into the component a bit more and found the `adjacent` slot. This (and the original margin it overrides) would affect anything in that slot. However, it's _only_ used in `filter-overflow-group` right now, which extends `OverflowGroupMixin` so this _doesn't_ affect it. It all still feels slightly fragile, but I can live with it.

Side note, `filter-overflow-group` has its own `margin-right` we ma want to rework as well.

Reply:
I did see that yeah -- agreed on the fragility.